### DISCUSSION TITLE
prisma generate into src directory

### DISCUSSION THREAD

**Discussion by tkroll** [TypedProject/ts-express-decorators#1904] [Q&A]
Possible to `npx prisma generate` into _src_ without it rewriting the entire directory?

**Comment by Romakita** [comment]
I don’t know. It’s question for prisma team. Not me ;)

**Reply by tkroll** [reply]
Fair enough. Haha.

Getting the same error as [here](https://github.com/tsedio/tsed/discussions/1844).
Trying to find a nicer place for those files.

**Reply by Romakita** [reply]
It’s a generator. By principe it’s normal to clear the directory where the code is generated. This is why by default the code is generated under a hidden directory, to prevent any change by a developer. I’m responsible if you generate the code in src. You should generate the code in a separated directory (and shouln’t commit the code in your repo).
